public/css/index.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background: #1e1e25;
+}
+
+.form {
+  position: absolute;
+  left: 0;
+  right: 0;
+  background: #27293d;
+  height: 90%;
+  width: 100%;
+  margin: 20px auto;
+  padding: 15px;
+  border-radius: 1%;
+}
+
+.form__label {
+  color: white;
+}
+
+.form__input {
+  background: none;
+  border: #2a324e solid 1px;
+  border-radius: 15px;
+  padding: 13px;
+  color: white;
+  width: 80%;
+}
+
+.form__input:focus {
+  border-color: #fb00ff;
+  background: #1d253b;
+  outline: none;
+}
+
+.form__field {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 40px;
+  position: relative;
+}
+
+.btn {
+  text-align: center;
+  padding: 10px 50px;
+  border-radius: 10px;
+  position: absolute;
+  color: white;
+  border: 1px solid gray;
+}
+
+.btn__field {
+  background: none;
+  border: none;
+  width: 10%;
+  cursor: pointer;
+  position: absolute;
+  right: 5%;
+  top: 50%;
+}
+
+.btn-blue {
+  background: #305cf4;
+  bottom: 20px;
+}
+
+.btn-pink {
+  background: #d54fd7;
+  bottom: 20px;
+  right: 20px;
+}
```

Comment:
jak już BEMowe nazewnictwo, to raczej tu modyfikatory z "--". Dodatkowo nie powinieneś nazywać od kolorów - bo te mogą się w przyszłości zmienić, tylko od funkcji jaką pełni przycisk chociażby